index.html

```diff
@@ -312,6 +518,63 @@
         wake lock application if they determine that the remaining battery
         capacity is low.
       </p>
+      <div class="note">
+        This section needs revision considering the changes in the API. In
+        particular, side communication channel possibility needs to be
+        re-evaluated as wake lock state is now observable.
+      </div>
```

Comment:
maybe raise an issue to make sure we don't lose track of it?

at least, change `<div class=note>` to `<div class=issue>`